src/arraytraits.rs

```diff
@@ -79,6 +81,88 @@ impl<S, S2, D> PartialEq<ArrayBase<S2, D>> for ArrayBase<S, D>
     }
 }
 
+impl<S, S2, D> PartialOrd<ArrayBase<S2, D>> for ArrayBase<S, D>
+    where D: Dimension,
+          S: Data,
+          S2: Data<Elem = S::Elem>,
+          S::Elem: PartialOrd,
+{
+    /// Return `None` if the array shapes `self` and `rhs` are not equal.
```

Comment:
It could be the doc on partialeq is so old it's from the time when ruatdoc did show...